lessons/Early-Math/translating/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+= Translating Between Words and Math
+
+@description{Students learn to model arithmetic expressions in words with a visual tool known as "Circles of Evaluation".}
+
+@lesson-prereqs{}
+
+@keywords{Circle of Evaluation, mathematical expression, equation}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Model an arithmetic expression using @vocab{Circles of Evaluation}.
+* Translate between expressions in words, Circles of Evaluation, and mathematical expressions.
+* Recognize when expressions in words are ambiguous.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's analyze the structure of expressions in words and numbers using Circles of Evaluation.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* This activity involves a card sort. The easiest way to prepare for the card sort is to give each pair of students an envelope containing the three sets of cards. Keep each set (Expressions, Circles of Evaluation, and Words) together with rubber bands or paperclips.
+
+| Key Points For The Facilitator
+|
+* This lesson may challenge students' ideas of math as a subject that is entirely black and white. This theme - that there are oftentimes a variety of completely valid ways of seeing a seemingly simple problem - will emerge again.
+|===
+
+== Circles of Evaluation @duration{30 minutes}
+
+=== Overview
+Students match Circles of Evaluation to mathematical expressions, and then they consider how mathematical expressions in words map onto Circles of Evaluation.
+
+=== Launch
+
+Give each pair of students an envelope and explain that it contains @handout{pages/three-sets-cards.adoc, "three sets of cards"}.
+
+[.lesson-instruction]
+- Look through the first set of cards. What do you notice?
+** _Students should notice that the first set of cards includes @vocab{mathematical expressions}. The expressions on these cards each include the number 15, the number 3, and an operator. Note that because these are expressions - not @vocab{equations} - they do not include an equal sign._
+- Now, look through the second set of cards. What do you notice?
+** _Students should observe that each card includes a circle. They may also notice the position of the numbers and operator within the circle._
```

Comment:
I think we can do better here. First, the circles of evaluation aren't actually circular, so I highly doubt that students will actually describe them as circles (unless you want to make the card sort a different way). Could we speak more broadly? I'm not sure I have the best words to offer up here, but perhaps the numbers are inside a diagram? or a shape?